tests/test_backend.py

```diff
@@ -7,14 +7,14 @@
 from mkosi.backend import Distribution, PackageType, set_umask
 
 
-def test_distribution():
+def test_distribution() -> None:
     assert Distribution.fedora.package_type == PackageType.rpm
     assert Distribution.fedora is Distribution.fedora
-    assert Distribution.fedora is not Distribution.debian
+    assert Distribution.fedora.package_type is not Distribution.debian.package_type
```

Comment:
[Mypy dislikes Enum comparisons](https://github.com/python/mypy/issues/9005). Since the testing is quite trivial I just compared different package types to assure it is actually different.